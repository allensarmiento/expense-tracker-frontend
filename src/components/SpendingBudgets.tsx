import BannerSection from './BannerSection';
import BannerTitle from './BannerTitle';

const SpendingBudgets = () => {
  return (
    <BannerSection>
      <BannerTitle>Spending Budgets</BannerTitle>
    </BannerSection>
  );
};

export default SpendingBudgets;
