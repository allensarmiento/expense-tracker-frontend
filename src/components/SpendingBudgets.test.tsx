import { shallow } from 'enzyme';
import SpendingBudgets from './SpendingBudgets';

it('renders without crashing', () => {
  expect(shallow(
    <SpendingBudgets />
  )).toMatchSnapshot();
});
