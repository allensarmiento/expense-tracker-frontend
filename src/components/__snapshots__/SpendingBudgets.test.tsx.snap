// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<BannerSection>
  <BannerTitle>
    Spending Budgets
  </BannerTitle>
</BannerSection>
`;
